tests/phpunit/unit/api/html/beansIsHtmlDevMode.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Tests for _beans_is_html_dev_mode()
+ *
+ * @package Beans\Framework\Tests\Unit\API\HTML
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Unit\API\HTML;
+
+use Beans\Framework\Tests\Unit\API\HTML\Includes\HTML_Test_Case;
+use Brain\Monkey;
+
+require_once __DIR__ . '/includes/class-html-test-case.php';
+
+/**
+ * Class Tests_BeansIsHtmlDevMode
+ *
+ * @package Beans\Framework\Tests\Unit\API\HTML
+ * @group   unit-tests
+ * @group   api
+ */
+class Tests_BeansIsHtmlDevMode extends HTML_Test_Case {
+
+	/**
+	 * Test _beans_is_html_dev_mode() should return false when the option does not exist.
+	 */
+	public function test_should_false_when_option_does_not_exist() {
+		$this->assertFalse( defined( 'BEANS_HTML_DEV_MODE' ) );
```

Comment:
I'm not sure why this assertion is here.

Reply:
There are 2 different results from this function:

1. That it passed the constant when defined, i.e. that's for legacy code.
2. Or when not defined, it goes to the database and returns the value if it exists.  If no, it returns the default.

Reply:
Right - but there's no test (or at least wasn't) that this function returns the value correctly when the constant is defined.